src/zcl/definition/tstype.ts

```diff
@@ -21,6 +21,59 @@ interface ParameterDefinition extends TsType.Parameter {
     type: DataType | BuffaloZclDataType;
 }
 
+interface AttributeDefinition {
+    ID: number;
+    type: DataType;
+    manufacturerCode?: number;
+}
+
+interface ClusterDefinition {
+    ID: number;
+    manufacturerCode?: number;
+    attributes: {[s: string]: AttributeDefinition};
+    commands: {
+        [s: string]: CommandDefinition;
+    };
+    commandsResponse: {
+        [s: string]: CommandDefinition;
+    };
+}
+
+interface CommandDefinition {
+    ID: number;
+    parameters: ParameterDefinition[];
+    response?: number;
+}
+
+type ClusterName = (
```

Comment:
note to self: when fixing the ikea stuff, remove the ikea custom clusters from here after removing them from zcl/definition/cluster.ts